buildshiprun/handler.go

```diff
@@ -59,7 +59,7 @@ func Handle(req []byte) string {
 		gatewayURL := os.Getenv("gateway_url")
 
 		// Replace image name for "localhost" for deployment
-		imageName = repositoryURL + imageName[strings.Index(imageName, ":"):]
+		imageName = repositoryURL + imageName[strings.Index(imageName, "/"):]
```

Comment:
Please break it out into a function with a unit test

Reply:
Ok